modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -31,7 +31,7 @@ For information about Cypher 5, see the link:https://neo4j.com/docs/cypher-manua
 To select the Cypher version of a query, prepend it with `CYPHER <language version>`. 
 
 Selecting `CYPHER 5` ensures that the query will be executed using the language as it existed at the time of the Neo4j 5.26 long-term support release.
-Any changes introduced in Neo4j 2025.03 or later will not affect the query.
+Any changes introduced after 5.26 will not affect the query.
```

Comment:
I will say that in theory this is correct, in practice not entirely XD

I know some things slipped though the cracks in the 2025.01 release I believe and then I know we on purpose added the SET DEFAULT LANGUAGE clauses to both cypher 5 and 25 but its not in 5.26 🙈 (and we will not document the Cypher 5 addition until Cypher 25 is available as well and then just add it added in Cypher 5 when Cypher 25 were released 🤷)